### PR TITLE
Improve micromamba transaction message

### PR DESCRIPTION
**Pull request by ruben-arts**
Before:
```bash
Transaction finished

To activate this environment, use:

    $ [micro]mamba activate <environment>

Or to execute a single command in this environment, use:

    $ [micro]mamba run -n <environment> mycommand
```
Examples of the result now:
```bash
$ time ./build/micromamba/micromamba create -c conda-forge --yes -n test python

# results in

Transaction finished

To activate this environment, use:

    micromamba activate test

Or to execute a single command in this environment, use:

    micromamba run -n test mycommand
    

$ time ./micromamba create -c conda-forge --yes -p ~/testing python

# results in, note the -p or -n

Transaction finished

To activate this environment, use:

   micromamba activate /home/rarts/testing

Or to execute a single command in this environment, use:

    micromamba run -p /home/rarts/testing mycommand
```

Notable improvements:
- If is_micromamba is true micromamba will be used as the executable
- Use no `$` to make the line double click + copy + paste.
- Use `-p` when using prefix, use `-n` for a named env.

### CODE REVIEW

File: libmamba/src/core/transaction.cpp [hunk 1]
```diff
@@ -1086,15 +1086,28 @@ namespace mamba
...
         {
-            // TODO: add executable name
-            // TODO: add environment name/prefix
             LOG_INFO << "Waiting for pyc compilation to finish";
             m_transaction_context.wait_for_pyc_compilation();
+
+            // Get the name of the executable used directly from the command.
+            std::istringstream iss(ctx.command_params.current_command);
+            std::string executable;
+            std::getline(iss, executable, ' ');
```
Comment (AntoinePrv) [inline libmamba/src/core/transaction.cpp:1095]: Could this be a method of the command?

**Comment by wolfv** [comment]
Unfortunately the `executable` won't work for `activate` because the activation is actually implemented on the shell level. So it has to be `micromamba activate ...`.

**Comment by AntoinePrv** [comment]
So maybe you can switch with `ctx.is_micromamba`?

**Comment by ruben-arts** [comment]
> Unfortunately the `executable` won't work for `activate` because the activation is actually implemented on the shell level. So it has to be `micromamba activate ...`.

Should we keep the run with the executable or keep the also move that to micromamba?

**Comment by wolfv** [comment]
I think the executable path can be long and weird so maybe just micromamba is better

**Review by Hind-M — APPROVED** [review]
Thanks!

**Pull request merged**